_headache.config

```diff
@@ -1,67 +0,0 @@
-################################################################################
```

Comment:
Please keep.

Reply:
Will do, but I'm curious, why would you want to keep this one and the other one ?

Reply:
They are configuration files for headache, I use them to update the header of files from time to time.